# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/PNecko.ipdl
# Commit: c684f79d8c67
# Full Hash: c684f79d8c673f21179fce00358d70845a9692c0
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Rename PWebrtcProxyChannel to PWebrtcTCPSocket. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45098
# ==============================================================================

diff -r 82af7b115f60 -r c684f79d8c67 netwerk/ipc/PNecko.ipdl
--- a/netwerk/ipc/PNecko.ipdl	Tue Sep 17 18:15:23 2019 +0000
+++ b/netwerk/ipc/PNecko.ipdl	Tue Sep 17 18:15:41 2019 +0000
@@ -26,7 +26,7 @@
 include protocol PStunAddrsRequest;
 include protocol PFileChannel;
 include protocol PClassifierDummyChannel;
-include protocol PWebrtcProxyChannel;
+include protocol PWebrtcTCPSocket;
 include protocol PSocketProcessBridge;
 include protocol PProxyConfigLookup;
 include protocol PDocumentChannel;
@@ -68,7 +68,7 @@
   manages PAltDataOutputStream;
   manages PStunAddrsRequest;
   manages PClassifierDummyChannel;
-  manages PWebrtcProxyChannel;
+  manages PWebrtcTCPSocket;
   manages PProxyConfigLookup;
   manages PDocumentChannel;
 
@@ -143,7 +143,7 @@
 
   async PStunAddrsRequest();
 
-  async PWebrtcProxyChannel(TabId tabId);
+  async PWebrtcTCPSocket(TabId tabId);
 
   async PProxyConfigLookup();
 